pypresence/payloads.py

```diff
@@ -32,16 +32,16 @@ def set_activity(cls, pid: int = os.getpid(),
                      activity: Union[bool, None] = True,
                      _rn: bool = True):
 
-        # They should already be an int because we give typehints, but some people are fucking stupid and use
+        # These should already be an int because we give typehints, but some people are fucking stupid and use
         # IDLE or some other stupid shit.
-        if start:
+        if type(start) == str:
             start = int(start)
-        if end:
+        if type(end) == str:
```

Comment:
If you're going to insist on type checking here, use `isinstance ` instead of `type()` please.
Additionally, this causes any other class that implements `__int__` to become unsupported here, which isn't something I think we should do.